### DISCUSSION TITLE
Module not found: Errors: Can't resolve '@theme/DocVersionSuggestions' and others

### DISCUSSION THREAD

**Discussion by ramsampath** [facebook/docusaurus#9226] [General]
Module not found: Error: Can't resolve '@theme/DocVersionSuggestions' 
Module not found: Error: Can't resolve '@theme/hooks/useDocs' 
Module not found: Error: Can't resolve '@theme/hooks/useUserPreferencesContext' 
Module not found: Error: Can't resolve '@theme/hooks/useLockBodyScroll' in 
Module not found: Error: Can't resolve '@theme/hooks/useWindowSize' in 
Module not found: Error: Can't resolve '@theme/hooks/useScrollPosition' in 
Module not found: Error: Can't resolve '@theme/IconArrow' in 
Module not found: Error: Can't resolve '@theme/IconMenu' in 

I am getting these errors .... I have tried to remove node_modules folder and tried again with yarn install and it still seems to give those errors in build.  Would appreciate any ideas on how to fix this..

Thank you
Ra.

**Comment by slorber** [comment]
You likely have swizzled some components and you need to maintain them yourself now.

`@theme/DocVersionSuggestions` does not exist anymore on newer versions of Docusaurus

You can erase your swizzlings and eventually reswizzle to reapply your customizations
